server/src/server.ts

```diff
@@ -52,20 +52,17 @@ let currentActiveTextDocument: TextDocument = TextDocument.create(
 )
 
 connection.onInitialize(async (params: InitializeParams): Promise<InitializeResult> => {
-  logger.level = params.initializationOptions?.loggingLevel ?? 'error'
+  logger.level = 'debug'

```

Comment:
I think you commited the loger level by mistake to debug instead of error

Reply:
done

Reply:
Now you removed the initializationOptions level that was passed. Just revert to the original line please.

Reply:
If you ammend the commit that brought this change directly rather than creating another commit it would look even more lean ;)

Reply:
`initializationOptions` is no longer passed: https://github.com/yoctoproject/vscode-bitbake/commit/fd2f0210d3417aecfca9d81a7d9795a724c67ab2